packages/expo/cli/api/signManifest.ts

```diff
@@ -0,0 +1,36 @@
+import { ExpoAppManifest, ExpoUpdatesManifest } from '@expo/config';
+import { JSONObject } from '@expo/json-file';
+
+import { fetchAsync } from './rest/client';
+import { ensureLoggedInAsync } from './user/actions';
+import { getActorDisplayName, getUserAsync } from './user/user';
+
+/** Sign an Expo Updates manifest to access secure features in sandboxed environments like Expo Go. */
+export async function signEASManifestAsync(manifest: ExpoUpdatesManifest): Promise<string> {
```

Comment:
Some requested renames:

```suggestion
export async function signExpoGoManifestAsync(manifest: ExpoUpdatesManifest): Promise<string> {
```